heltour/api_worker/views.py

```diff
@@ -21,10 +21,11 @@ def _get_lichess_api_token():
         return None
 
 def _do_lichess_api_call(redis_key, path, method, post_data, params, priority, max_retries, format,
-                         retry_count=0):
```

Comment:
If this is optional, shouldn't we make it actually optional by setting a default value to `None`?